tee-worker/omni-executor/.gitignore

```
@@ -3,6 +3,8 @@ target/
 .idea/
 **/*.bin
 
+storage
```

Thanks, I forgot to add it. The path was renamed to `storage_db`
```suggestion
storage_db
```